awx/ui/po/ja.po

```diff
@@ -6699,3 +6703,8 @@ msgstr "{{::state._hint}}"
 msgid "{{pageSize}}"
 msgstr "{{pageSize}}"
 
+#: client/src/instance-groups/container-groups/add-container-group.view.html:5
+msgid "This feature is currently in tech preview and is subject to change in a future release.  Click here for documentation."
```

Comment:
Generally we put these translations into memsource so that we don't regress when new translations are pulled down.  I'd remove this change from the PR.